docs/README.md

```diff
@@ -1,39 +1,38 @@
-# Documentation
+# Cosmos Cash
 
-How to use the Cosmos Cash Documentation.
+[![Go Reference](https://pkg.go.dev/badge/github.com/allinbits/cosmos-cash.svg)](https://pkg.go.dev/github.com/allinbits/cosmos-cash)
+[![build](https://github.com/allinbits/cosmos-cash/actions/workflows/ci.yaml/badge.svg?branch=main)](https://github.com/allinbits/cosmos-cash/actions/workflows/ci.yaml)
+[![codecov](https://codecov.io/gh/allinbits/cosmos-cash/branch/main/graph/badge.svg?token=NLT5ZWM460)](https://codecov.io/gh/allinbits/cosmos-cash)
+[![Libraries.io dependency status for GitHub repo](https://img.shields.io/librariesio/github/allinbits/cosmos-cash)](https://libraries.io/go/github.com%2Fallinbits%2Fcosmos-cash)
 
-- [Documentation](#documentation)
-    - [Overview](#overview)
-    - [Contributing](#contributing)
-    - [Reference](#reference)
+![](https://miro.medium.com/max/1000/1*8Wx44uvyJxpZUVS0WojMNw.png)
 
-## Overview
+### Summary
 
-In Cosmos Cash we use the  *Grand Unified Theory of Documentation* (David Laing) as described
-by [Divio](https://documentation.divio.com/) as a basis for our documentation strategy.
+Cosmos Cash is a protocol designed to be regulatory compliant that offers the same guarantees as traditional banking
+systems. Features that enable these guarantees are Know Your Customer (KYC), anti-money laundering (AML) tracking,
+Financial Action Task Force (FATF) travel rule, and identity management. Cosmos Cash uses a novel approach to identity
+management by leveraging W3C specifications for decentralized identifiers and verifiable credentials.
 
-This approach outlines four specific use cases for documentation:
+### Research paper
 
-* [Tutorials](./Tutorials/README.md)
-* [How-Tos](./How-To/README.md)
-* [Explanation](./Explanation/README.md)
-* [Reference](./Reference/README.md)
+For more information on the research behind the Cosmos Cash protocol, please look at the Cosmos Cash research paper:
```

Comment:
```suggestion
For more information on the research behind the Cosmos Cash protocol, see the Cosmos Cash research paper:
```

Reply:
oops let's not use please 
https://developers.google.com/style/tone#politeness-and-use-of-please